docs/getting-started/auth.adoc

```diff
@@ -1,15 +1,169 @@
 = AeroGear Services Auth SDK
 
-Mobile authentication SDK based on Keycloak
+Mobile authentication SDK based on link:http://www.keycloak.org/[Keycloak] using link:http://openid.net/connect/[OpenID Connect].
+
+Provides authentication features like access control and two factor authentication through Keycloak.
 
 == Adding dependency
```

Comment:
@aidenkeating I'd suggest swapping this section with the Usage section i.e. show the Usage first for including the Core SDK. Maybe even have this section as part of Usage?